projects/js-packages/components/components/record-meter-bar/index.tsx

```diff
@@ -106,6 +107,30 @@ const RecordMeterBar: React.FC< RecordMeterBarProps > = ( {
 					} ) }
 				</ul>
 			</div>
+			<table
+				className="screen-reader-text"
+				summary="This is the screen reader alternative data table for the record meter bar graphic."
+			>
+				<caption>{ __( 'Records for Record Meter Bar', 'jetpack' ) }</caption>
+				<tbody>
+					<tr>
+						<th scope="col">{ __( 'Record type', 'jetpack' ) }</th>
+						{ itemsToRender.map( ( { label } ) => {
+							return (
+								<th scope="col" key={ label }>
+									{ label }
+								</th>
+							);
+						} ) }
+					</tr>
+					<tr>
+						<th scope="row">{ __( 'Record Count', 'jetpack' ) }</th>
```

Comment:
```suggestion
						<th scope="row">{ __( 'Record count', 'jetpack' ) }</th>
```

Reply:
^ Sentence case to be consistent with "Record type" and also other headings in the dashboard.